viewlib.php

```diff
@@ -261,8 +261,15 @@ function bigbluebuttonbn_view_render_recordings(&$bbbsession, $enabledfeatures,
     );
     // If there are meetings with recordings load the data to the table.
     if ($bbbsession['bigbluebuttonbn']->recordings_html) {
+        $recordingshtml = '';
+        // If opencast recoding config is set and checks if opencast is available for the current course.
+        if ((boolean) \mod_bigbluebuttonbn\locallib\config::get('oc_recording')
```

Comment:
As I have written in the general feedback, I propose to add another admin setting which specifically controls if the recordings in Opencast are shown here or not.